src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -5,98 +5,512 @@
+
+    # Download compressed helm binary if not already present
+    if not os.path.isfile(download_location):
+        # Creating the helm folder if it doesnt exist
+        if not os.path.exists(download_dir):
+            try:
+                os.makedirs(download_dir)
+            except Exception as e:
+                logger.warning("Failed to create helm directory." + str(e))
+                return
+
+        # Downloading compressed helm client executable
+        try:
+            response = urllib.request.urlopen(requestUri)
+        except Exception as e:
+            logger.warning("Failed to download helm client.")
+            return
+
+        responseContent = response.read()
+        response.close()
+
+        # Creating the compressed helm binaries
+        try:
+            with open(download_location, 'wb') as f:
+                f.write(responseContent)
+        except Exception as e:
+            logger.warning("Failed to extract helm executable" + str(e))
+            return
+
+    # Extract compressed helm binary
+    if not os.path.isfile(install_location):
+        try:
+            shutil.unpack_archive(download_location, download_dir)
+            os.chmod(install_location, os.stat(install_location).st_mode | stat.S_IXUSR)
+        except Exception as e:
+            logger.warning("Failed to extract helm executable" + str(e))
+            return
+
+    return install_location
+
+
+def install_kubectl_client():
+    # Return kubectl client path set by user
+    try:
+
+        # Fetching the current directory where the cli installs the kubectl executable
+        home_dir = os.path.expanduser('~')
+        kubectl_filepath = os.path.join(home_dir, '.azure', 'kubectl-client')
+
+        try:
+            os.mkdir(kubectl_filepath)
+        except FileExistsError:
+            pass
+
+        operating_system = platform.system().lower()
+        # Setting path depending on the OS being used
+        if operating_system == 'windows':
+            kubectl_path = os.path.join(kubectl_filepath, 'kubectl.exe')
+        elif operating_system == 'linux' or operating_system == 'darwin':
+            kubectl_path = os.path.join(kubectl_filepath, 'kubectl')
+        else:
+            logger.warning(f'The {operating_system} platform is not currently supported for installing kubectl client.')
+            return
+
+        if os.path.isfile(kubectl_path):
+            return kubectl_path
+
+        # Downloading kubectl executable if its not present in the machine
+        get_default_cli().invoke(['aks', 'install-cli', '--install-location', kubectl_path])
+        # Return the path of the kubectl executable
+        return kubectl_path
+
+    except Exception as e:
+        logger.warning("Unable to install kubectl. Error: " + str(e))
+        return
+
+
 class Connectedk8sScenarioTest(LiveScenarioTest):
 
-    def test_connectedk8s(self):
+    @live_only()
```

Comment:
May I ask why these tests marked as `@live_only()`?

Reply:
I also have the same problem.

Reply:
- If you want to run recording tests on every commit, please remove the @live_only decorator and we will run the recording tests after every commit. (Please remember to upload the recording file generated by the live test)
- Otherwise none of the following tests will actually be executed：
![image](https://user-images.githubusercontent.com/18628534/219528190-7b72b5a5-415a-412c-ba5a-654699517ba3.png)

Reply:
@wangzelin007 if you remember we earlier attempted to try the recording mode but it didnt work out because of some issue in our connectedk8s connect code ( we are fetching tenant-id from cache because of which the tests were failing in recording mode) so for now we have decided to go with live mode only , we know this will not run as integration tests but till the time we find a solution for our command, we will make this a practice from our end that the member that raises the PR will trigger the CLI testing pipeline

@sirireddy12 if you have anything to add